{"name": "test_edit_general_information", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <test_login.Test001Login object at 0x7fa552683eb0>\nsetup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"5ece3d8871c7641ec8c975f07f6c3562\")>\n\n    def test_edit_general_information(self,setup):\n        self.driver = setup\n>       gb = GeneralInfo(self.driver)\n\ntestCases/test_login.py:87: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npageObjects/ProjectDetailsForm/GeneralInfo.py:24: in __init__\n    self.goto()\npageObjects/ProjectDetailsForm/GeneralInfo.py:27: in goto\n    link = utils.explicit_wait_present(self.driver,self.general_info_xpath)\nUtils/ComplexElement.py:14: in explicit_wait_present\n    element = wait.until(EC.presence_of_element_located((By.XPATH,locator)))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"5ece3d8871c7641ec8c975f07f6c3562\")>\nmethod = <selenium.webdriver.support.expected_conditions.presence_of_element_located object at 0x7fa55258b400>, message = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n../../../.local/lib/python3.8/site-packages/selenium/webdriver/support/wait.py:80: TimeoutException"}, "start": 1629274521062, "stop": 1629274531122, "uuid": "b9762198-25f2-4f8e-93c9-0bacbdf739df", "historyId": "2a3c845c0c03891894f72d451a907107", "testCaseId": "93a7d1ad17ebbf608b1750e7928ccb04", "fullName": "testCases.test_login.Test001Login#test_edit_general_information", "labels": [{"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test001Login"}, {"name": "host", "value": "bipin-Nitro-AN515-54"}, {"name": "thread", "value": "81224-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_login"}]}